### ISSUE TITLE
Where are settings?

### ISSUE THREAD

**Issue by colegota** [mikifus/padland#38]
Sorry if this is a stupid question but not sure if I'm stupid or had some trouble at my end, but I can't find how to reach settings to create a new server.

I'm using 1.3.6 Fdroid version in a CyanogenMod 11 (Android 4.4).

As you can see at attached image I just have two icons for new group and new pad. This one ask me for name and server but it only offer a list with predefined path.
There are no menu icons at all, the menu button does nothing and no way to slide from side or other ways to show a Preferences menu.

Also as you can see at the screenshot, the path for riseup pad created has the url included twice and does not work.

![image-1373272175](https://user-images.githubusercontent.com/5110784/30772217-6aa39544-a057-11e7-81b0-e582a534427f.jpg)

Regards!
Colegota

**Comment by mikifus** [comment]
Hi,

TL;DR: top right corner, cogwheel icon in the toolbar.

It's in the top right corner, on the toolbar, after the title. Your Android version is not showing the icons correctly (they should be white) because of a bad implementation by Google. I didn't bother making compatibility as it requires extra images and code that will increase the app size and complexity. Also I was thinking about a redesign using the standard left sliding bar in a near future.

You are welcome to do it yourself if you find good way. Another solution is to upgrade your system, as you are using CyanogenMod that would be the best for now.

Please tell me if it works that way.

**Comment by mikifus** [comment]
Hi,

Sorry for the delay. I'll close this as won't fix because rebuilding the design will do. I did fix the tooltip, it was wrong in all languages, will be corrected on next version. Thanks for the offer on translations, maybe we'll need them at some point.
